### PR TITLE
added notifications table

### PR DESCRIPTION
**Pull request by mly32**
### Summary 

Added notifications table.

image

image

image

### Test Plan 

Try running on different pages.

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: frontend/src/components/Notification/Notification.tsx [hunk 1]
```diff
@@ -1,18 +1,84 @@
...
+
+  const onClick = () => {
+    console.log('clik');
+  };
+
+  return (
+    <Popup
+      trigger={
+        <button className={styles.bell} onClick={onClick}>
+          <span className={styles.span} />
```
Comment (mg2239) [inline frontend/src/components/Notification/Notification.tsx:51]: what are the spans for?

File: frontend/src/components/Notification/Notification.tsx [hunk 2]
```diff
@@ -1,18 +1,84 @@
...
-    )}
-  </div>
-);
+type Message = {
+  time: Date;
+  title: string;
+  body: string;
+};
+
+const Notification = () => {
```
Comment (mg2239) [inline frontend/src/components/Notification/Notification.tsx:15]: the notification components is dismounted and remounted on page change, which resubscribes the user. ideally wed want them to only subscribe once, would it be possible to either create a shared header component across routes, or move notification state to a context?

**Review by mg2239 — CHANGES_REQUESTED** [review]
looks good! just a couple changes mentioned in the comments, and also id like the notifications to be better aligned to styles, such as profile picture (can be placeholder for now), date formatting as month day rather than mm/dd/yyyy, only showing one line for notifications, etc. 

for the clickable "view" link you dont need to implement functionality right now, but once the ride info can be sent through notifications it should navigate to the schedule page and change the schedule date to the ride date

File: frontend/src/components/Notification/Notification.tsx (hunk 2 above)
Comment (mly32) [reply frontend/src/components/Notification/Notification.tsx:17]: Right, this is pretty important. I'll implement this in a different PR.

File: frontend/src/components/Notification/Notification.tsx (hunk 1 above)
Comment (mly32) [reply frontend/src/components/Notification/Notification.tsx]: They were for spacing but are no longer necessary. I have removed them.

**Review by mg2239 — APPROVED** [review]
nice work!

**Pull request merged**